src/pages/[slug]/index.tsx

```diff
@@ -445,6 +456,13 @@ const ReadingRoomPage: NextPage = () => {
 		<>
 			<MediaCardList
 				items={media?.items
+					.filter((mediaObject: MediaInfo) => {
+						if (query.format && query.format !== 'all') {
+							return mediaObject.dcterms_format === query.format;
+						} else {
+							return true;
+						}
+					})
```

Comment:
Avoid filtering front-end as it messes with pagination-UX

Reply:
yes, this isn't for the real results, but for the intermediate state between switching tabs

when you are on the "all" tab, you load some results (video + film + audio)
when you click on the video tab, you want to do a new query, without losing your tab count data
but if i don't filter the results, you'll see a flash of the old audio items in the video tab, which looks very wrong.

This avoids that flash and also keeps the tab count giving it a smoother and seemingly faster result